src/htr2hpc/train/data.py

```diff
@@ -63,7 +58,28 @@ def get_segmentation_data(
             )
         )
 
-    # gather base lines
+    # recognition training requires transcription text content
+    # if a transcription id is specified, retrieve transcription content
+    text_lines = {}
+    if transcription_id:
+        # API could have multiple pages of transcription lines;
+        # loop until all pages of results are consumed
+        while True:
+            transcription_lines = api.document_part_transcription_list(
+                document_id, part_id, transcription_id
+            )
+            for text_line in transcription_lines.results:
+                # Each transcription line includes a line id,
+                # transcription id, and text content.
+                # Add to dict so we can lookup content by line id
+                text_lines[text_line.line] = text_line.content
+            # if there is another page of results, get them
+            if transcription_lines.next:
+                transcription_lines = transcription_lines.next_page()
```

Comment:
This is not functioning correctly -- I get the following error for a part whose Line Transcription List has multiple pages:

```
htr2hpc/src/htr2hpc/api_client.py:57 in next_page

54 │   │   │   # convert result type (e.g. "list_model") into api method  
55 │   │   │   # (this may be brittle...)   
56 │   │   │   single_rtype = self.result_type.replace("list", "").strip( 
57 │   │   │   list_method = getattr(self.api, f"{single_rtype}_list") 
58 │   │   │   return list_method(page=next_page_num) 

AttributeError: 'eScriptoriumAPIClient' object has no attribute 
'transcription_line_list'
```

See for example https://test-htr.lib.princeton.edu/api/documents/27/parts/54422/transcriptions/

Reply:
hahaha wow, even says in my comments that this approach might be brittle. sure enough! 😂